src/server/middleware/twitter.middleware.ts

```diff
@@ -411,3 +592,32 @@ function getUserIdFromCredential(credential: firebase.auth.OAuthCredential) {
   const match = credential.accessToken?.match('[0-9]+');
   return match && match.length ? match[0] : null;
 }
+
+// Format a millisecond-based timestamp into the yyyymmddhhmm date format
+// suitable for the Enteprise Twitter API, as defined in:
+// https://developer.twitter.com/en/docs/tweets/search/api-reference/enterprise-search
+function formatTimestamp(ms: number): string {
+  const date = new Date(ms);
+  const MM = date.getUTCMonth() + 1; // getMonth() is zero-based
```

Comment:
Optional: Personally I would name these "month", "date", "hours", and "minutes" because it's much more readable for the code below in my opinion, and also less confusing if the reader doesn't know date formatting syntax. I see why you include them like this here, but the info about the format is already in the docstring (capitalization doesn't seem to match though). I would instead clarify what 'yyymmddhhmm' stands for in the docstring, or link to docs on date formatting for those who don't know it.